content/docs/introduction/faq.md

```diff
@@ -25,7 +25,9 @@ The main Prometheus server runs standalone and has no external dependencies.
 Yes, run identical Prometheus servers on two or more separate machines.
 Identical alerts will be deduplicated by the [Alertmanager](https://github.com/prometheus/alertmanager).
 
-The Alertmanager cannot currently be made highly available, but this is a goal.
+For [high availability of the Alertmanager](https://github.com/prometheus/alertmanager#high-availability),
+you can run multiple instances in a Mesh cluster and configure the Prometheus
```

Comment:
now you should link to Mesh rather than having this random capitalised word in here :)